docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -73,166 +71,6 @@ of which the prefix is changed from *spring.cloud.azure.eventhubs* to *spring.cl
 | boolean
 | Whether an Azure Event Hubs is enabled.
 
-|*spring.cloud.azure.eventhubs*.credential.*
```

Comment:
![image](https://user-images.githubusercontent.com/31124698/150900294-2421a63b-fee1-40dc-9750-52d36ecdc51e.png)

Reply:
`spring.cloud.azure.eventhubs.event-hub-name` and `spring.cloud.azure.eventhubs.is-shared-connection` are not useful when configuring connections for a binder.